share/nitweb/stylesheets/nitweb.css

```diff
@@ -29,8 +29,6 @@ a {
 	border: 1px solid #1E9431;
 }
 
-.card, .card-body { overflow: hidden; }
-
```

Comment:
:heavy_check_mark: in [#2177](https://github.com/nitlang/nit/issues/2177#issuecomment-227482006)